.cascade-code/left-nav-refactor/_components/left_nav/left_nav_no_siblings_format

```diff
@@ -0,0 +1,73 @@
+## Format used by index pages where siblings don't want to be shown.
+## This loads all children of the index page (folders and pages).
```

Comment:
If each format is intended to work with a specific block, maybe we should as a convention note that in header comment.